src/app/clusters/color-control-server/color-control-server.cpp

```diff
@@ -1656,6 +1664,16 @@ Status ColorControlServer::stepHueCommand(EndpointId endpoint, HueStepMode stepM
     VerifyOrReturnValue(stepMode != HueStepMode::kUnknownEnumValue, Status::InvalidCommand);
     VerifyOrReturnValue(stepSize != 0, Status::InvalidCommand);
 
+    // Command Parameters constraint checks:
+    if (isEnhanced)
+    {
+        VerifyOrReturnValue(transitionTime <= kMaxtransitionTime, Status::ConstraintError);
+    }
+    else
+    {
+        VerifyOrReturnValue(transitionTime <= UINT8_MAX, Status::ConstraintError);
```

Comment:
```suggestion
        VerifyOrReturnValue(transitionTime <= std::numeric_limits<uint8_t>::max(), Status::ConstraintError);
```

Reply:
isn't that the same thing but shorter ?

Reply:
In practice, it does not really matter since the only caller is passing in a `uint8` anyway, so this is just extra but somewhat pointless code....  Could just document why in the non-enhanced case we don't need a range check, if desired.